packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md

```diff
@@ -0,0 +1,343 @@
+			"hostname": "harvard",
+			"id": "94af8742-42f9-4560-9345-8eec11d6fded",
+			"instance_details": null,
+			"last_seen": "0001-01-01T00:00:00Z",
+			"managed": true,
+			"os": "Ubuntu 16.04",
+			"os_type": "linux",
+			"project_name": "the-sound-and-the-fury",
+			"registered_at": "0001-01-01T00:00:00Z",
+			"services": [
+				"ssh"
+			],
+			"sftd_version": "1.44.4",
+			"ssh_host_keys": null,
+			"state": "INACTIVE",
+			"team_name": "william-faulkner"
+		},
+		{
+			"access_address": null,
+			"alt_names": null,
+			"bastion": null,
+			"broker_host_certs": null,
+			"canonical_name": null,
+			"cloud_provider": null,
+			"deleted_at": "0001-01-01T00:00:00Z",
+			"hostname": "jefferson",
+			"id": "be241b5e-2bc2-4c19-8300-f2d8f21fbe71",
+			"instance_details": null,
+			"last_seen": "0001-01-01T00:00:00Z",
+			"managed": true,
+			"os": "Ubuntu 16.04",
+			"os_type": "linux",
+			"project_name": "the-sound-and-the-fury",
+			"registered_at": "0001-01-01T00:00:00Z",
+			"services": [
+				"ssh"
+			],
+			"sftd_version": "1.44.4",
+			"ssh_host_keys": null,
+			"state": "INACTIVE",
+			"team_name": "william-faulkner"
+		}
+	]
+}
+```
+### Fetch Team settings
+
+<ApiOperation method="GET" url="https://app.scaleft.com/v1/teams/${team_name}/settings" />
+Fetch Team-level settings for a specific Team, such as authentication and enrollment details.
+
+This endpoint requires one of the following roles: `access_admin`, `instance_admin`, `access_user`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+This endpoint has no request body.
+
+#### Response body
+This endpoint returns an object with the following fields and a `200` code on a successful call.
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `approve_device_without_interaction`   | boolean | If enabled, ASA will auto-approve devices for ASA Users authenticated into this Team. |
+| `client_session_duration`   | number | Configure client session to be between 1 hour and 25 hours. |
+| `post_device_enrollment_url`   | string | If configured, the URL a ASA User will be directed to after enrolling a device in ASA. |
+| `post_login_url`   | string | If configured, this is the URL a ASA User who has not recently been authenticated will be directed to after being validated by their IDP in the course of getting credentials. |
+| `post_logout_url`   | string | If configured, this is the URL a ASA User will be redirected to after logging out. |
+| `reactivate_users_via_idp`   | boolean | If a disabled or deleted ASA User is able to authenticate via the IDP, their ASA user will be re-enabled. |
+| `team`   | string | The name of the Team that is configured with the contained settings. |
```

Comment:
```suggestion
| `team`   | string | The name of the Team that is configured with the contained settings |
```
What does "contained" mean? I don't think all readers are going to know what "contained" means. Can we make this a bit more clear?